admin_manual/configuration_files/files_locking_enabling.rst

```diff
@@ -16,4 +17,28 @@ underlying filesystem.
 The Files Locking app has no configuration options; all you need to do is 
 enable or disable it on your Apps page.
 
-.. figure:: ../images/files_locking_app.png
\ No newline at end of file
+.. figure:: ../images/files_locking_app.png
+
+We recommend using `Redis <http://redis.io/>`_ as your ownCloud memcache when 
+you enable the Files Locking app. Memcached, the popular distributed memory 
```

Comment:
They are both about concurrency.

The difference is that files_locking use flock on filesystem level.

The new experimental locking uses a custom-lock implementation based on redis, and locks on View level. This makes it so that we can have "bigger" transactions during a lock.